alicloud/resource_alicloud_nat_gateway_test.go

```diff
@@ -179,6 +179,14 @@ func TestAccAlicloudNatGatewayBasic(t *testing.T) {
 					}),
 				),
 			},
+			{
+				Config: testAccNatGatewayConfig_type(rand),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{
+						"name": fmt.Sprintf("tf-testAccNatGatewayConfig%d_change", rand),
+					}),
+				),
+			},
 			{
```

Comment:
If `instance_charge_type` can not be updated, this step should be removed, and the its check should be added into other steps.